backend/python_init_script/requirements.txt

```diff
@@ -1,4 +1,4 @@
-appwrite
+appwrite==0.5.1
```

Comment:
What was your reason to require a specific version of Appwrite? Security purposes?

Reply:
no, compatibility. `create_collection` i.e, requires extra `uid` in input parameters. I didn't want to deal with this, so lock the version in this script would be best fit.